NuGet.Config

```diff
@@ -8,5 +8,10 @@
     <clear />
     <!-- Warning: Do not add/change the NuGet feeds here. To be compliant this repo must only rely on this single ADO feed. -->
     <add key="react-native" value="https://pkgs.dev.azure.com/ms/react-native/_packaging/react-native-public/nuget/v3/index.json" />
+
```

Comment:
We'll have to watch if this triggers the compliance feed detection during publish - I'm not sure if the tooling is smart enough to see that it's commented out.